doc/common/glossary.md

```diff
@@ -20,94 +20,136 @@ To help guide the reader, this glossary provides an introduction to the terminol
- >_*Note:*_ The NFVI can span across many locations, e.g. places where data centres or edge nodes are operated. The network providing connectivity between these locations is regarded to be part of the cloud infrastructure. **NFVI** and **VNF** are the top-level conceptual entities in the scope of Network Function Virtualisation. All other components are sub-entities of these two main entities.
-- **Cloud Infrastructure**: A generic term covering **NFVI**, **IaaS** and **CaaS** capabilities - essentially the infrastructure on which a **Workload** can be executed.
+- **Cloud Infrastructure:** A generic term covering **NFVI**, **IaaS** and **CaaS** capabilities - essentially the infrastructure on which a **Workload** can be executed.
 >_*Note:*_ **NFVI**, **IaaS** and **CaaS** layers can be built on top of each other. In case of CaaS some cloud infrastructure features (e.g.: HW management or multitenancy) are implemented by using an underlaying **IaaS** layer.
-- **Network Function (NF)**:  functional block or application within a network infrastructure that has well-defined external interfaces and well-defined functional behaviour.
-  - Within **NFV**, A **Network Function** is implemented in a form of **Virtualised NF** or a **Cloud Native NF**.
-- **Network Service (NS)**: composition of **Network Function**(s) and/or **Network Service**(s), defined by its functional and behavioural specification, including the service lifecycle.
-- **Virtual Network Function (VNF)**: a software implementation of a **Network Function**, capable of running on the **Cloud Infrastructure**.
-  - **VNF**s are built from one or more VNF Components (**VNFC**) and, in most cases,  the VNFC is hosted on a single VM or Container.
-- **Cloud Native Network Function (CNF)**: A cloud native network function (CNF) is a cloud native application that implements network functionality. A CNF consists of one or more microservices and has been developed using Cloud Native Principles including immutable infrastructure, declarative APIs, and a “repeatable deployment process”.
+
+- **Cloud Infrastructure Profile:** The combination of the Cloud Infrastructure Software Profile and the Cloud Infrastructure Hardware Profile that defines the capabilities of the Cloud Infrastructure.
+
+- **Cloud Infrastructure Software Configuration:** a set of settings (Key:Value) that are applied/mapped to **cloud infrastructure** SW deployment.
+
+- **Cloud Infrastructure Software Profile:** defines the behaviour, capabilities and metrics provided by a cloud infrastructure Software Layer
+
+- **Cloud Native Network Function (CNF):** A cloud native network function (CNF) is a cloud native application that implements network functionality. A CNF consists of one or more microservices and has been developed using Cloud Native Principles including immutable infrastructure, declarative APIs, and a “repeatable deployment process”.
```

Comment:
```suggestion
- **Cloud Native Network Function (CNF):** A cloud native network function (CNF) is a cloud native application that implements network functionality. A CNF consists of one or more microservices that have been developed using Cloud Native Principles including immutable infrastructure, declarative APIs, and a “repeatable deployment process”.
```

Reply:
I'd argue this doesn't need changing, as the original text refers to the CNF being `developed using Cloud Native Principles....` as opposed to the micro services.  Not a major issue either way, but I think we took the wording from the work we'd done in the TUG.

Reply:
@tomkivlin then teh sentence needs correcting

Reply:
Do we want to say that the microservices have been developed using CNF principles or that the CNF has been developed using CNF principles?
What should be the corrected version?

Reply:
@EditKoselak I think @tomkivlin is suggesting, "that the CNF has been developed using cloud native principles".  My issue is that if the CNF consists of microservices and that the CNF has been developed using cloud native principles then the constituent microservices should also have been developed using cloud native principles.

Please do whatever you think is right.

Reply:
I agree @pgoyal01 that the microservices should also be developed following cloud native principles. Meaning all **layers** of a network function (or single network service end point) claiming to be cloud native should be implemented following those principles.

Ideally all layers of a workload would be implemented using cloud native principles from the smallest component all the way up.  In the end your workload is a composition of of all the network service end points.  If every network service end point is implemented cloud natively then the entire workload would be cloud native.

Reply:
Thanks, we have attempted to resolve this issue, please check the updated version:)